deploy/.env.example

```diff
@@ -35,6 +38,13 @@ FOREIGN_GAS_PRICE=10000000000
 #for bridge erc_to_erc and erc_to_native mode
 ERC20_TOKEN_ADDRESS=
 
+#Optional Implemetation addresses for ERC_TO_ERC_MULTIPLE mode
+HOME_BRIDGE_VALIDATORS_IMPLEMENTATION_ADDRESS=0x
+HOME_BRIDGE_IMPLEMENTATION_ADDRESS=0x
+FOREIGN_BRIDGE_VALIDATORS_IMPLEMENTATION_ADDRESS=0x
+FOREIGN_BRIDGE_IMPLEMENTATION_ADDRESS=0x
+DOUBLE_PROXY_IMPLEMENTATIONS=
```

Comment:
Does it make sense to provide opportunity to disable this? Would it be worth to use double proxy always if ERC_TO_ERC_MULTIPLE configuration is used?

Reply:
> Does it make sense to provide opportunity to disable this?
In my opinion, yes.
Different bridges (implementation version) can be applied here.